src/main/kotlin/no/nav/syfo/personstatus/domain/PersonOversiktStatus.kt

```diff
@@ -125,6 +128,9 @@ fun PersonOversiktStatus.hasActiveBehandlerdialogOppgave(): Boolean {
         this.behandlerdialogAvvistUbehandlet
 }
 
+private fun PersonOversiktStatus.hasFriskmeldingTilArbeidsformidling(today: LocalDate): Boolean =
+    this.friskmeldingtilarbeidsformidlingFom != null && today.isBeforeOrEqual(this.friskmeldingtilarbeidsformidlingFom)
```

Comment:
For meg føles det mer naturlig å bare lage en `today` her! Det er rart å kreve at konsumenter sender inn et argument som uansett skal være likt for alle kallene.

Det er vel ikke noe særlig kostnad å gjøre det mange ganger. Og så lenge man forholder seg til dag og ikke sekunder, er det ikke noe særlig risiko for at `today` blir annerledes for siste element i en liste enn det første.